ci/tests.yml

```diff
@@ -0,0 +1,22 @@
+include:
+- remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
+
+stages:
+- test
+
+run tests:
+  extends: .container-runner-daint-gpu
+  stage: test
+  image: $CSCS_REGISTRY_PATH/gt4py/gt4py-ci:$PYVERSION
+  rules:
+  - if: $CI_PIPELINE_SOURCE == "parent_pipeline"
+  script:
+  - cd /gt4py.src
+  - python -c "import cupy"
+  - tox run -e $SUBPACKAGE-$PYVERSION_PREFIX-$BACKEND-cuda112
```

Comment:
Just discovered we are not running cpu at all. We should discuss if that is what we want.

Reply:
Yes, we can reconsider now that the queue times etc are not as problematic.

Reply:
I will create another experimental branch running all the tests to collect data on how long it would take with this new system.